Modules/_elementtree.c

```diff
@@ -495,12 +495,12 @@ element_resize(ElementObject* self, Py_ssize_t extra)
              * "children", which needs at least 4 bytes. Although it's a
              * false alarm always assume at least one child to be safe.
              */
-            children = PyObject_Realloc(self->extra->children,
+            children = PyMem_Realloc(self->extra->children,
```

Comment:
The following line was previously aligned with the `(`. It probably also fits on one line now.

Reply:
Thanks for pointing that out. I'll fix it before merging!